factory/ideas.php

```diff
@@ -205,36 +230,31 @@ public function get_ideas($number = 10, $sort = 'date', $sort_direction = 'DESC'
 	 */
 	public function get_idea($id)
 	{
-		$sql = 'SELECT *
-			FROM ' . $this->table_ideas . "
-			WHERE idea_id = $id";
+		$sql_array = array(
+			'SELECT'		=> 'i.*, d.duplicate_id, t.ticket_id, r.rfc_link',
+			'FROM'			=> array($this->table_ideas => 'i'),
+			'LEFT_JOIN'		=> array(
+				array(
+					'FROM'	=> array($this->table_duplicates => 'd'),
+					'ON'	=> 'i.idea_id = d.idea_id',
+				),
+				array(
+					'FROM'	=> array($this->table_tickets => 't'),
+					'ON'	=> 'i.idea_id = t.idea_id',
+				),
+				array(
+					'FROM'	=> array($this->table_rfcs => 'r'),
+					'ON'	=> 'i.idea_id = r.idea_id',
+				),
+			),
+			'WHERE'			=> 'i.idea_id = ' . (int) $id,
+		);
+
+		$sql = $this->db->sql_build_query('SELECT', $sql_array);
```

Comment:
But there are a lot of operations that update the idea... Voting and mod operations.